string source = rogr
string target = armcamera
time timestamp
bool camera_on

